lib/Cake/Controller/Component/FlashComponent.php

```diff
@@ -38,6 +38,7 @@ class FlashComponent extends Component {
 		'key' => 'flash',
 		'element' => 'default',
 		'params' => array(),
+		'clear' => false
```

Comment:
I don't know what you agreed on, but it looks like a trailing comma is not being used in similar instances the other components currently